schema.ts

```diff
@@ -1383,7 +1383,18 @@ export interface ObjectSchemaDefinition<K extends string, L extends string>
    * TODO(ebo): Unhide this
    * @hidden
    */
-  versionProperty? : PropertyIdentifier<K>;
+  versionProperty?: PropertyIdentifier<K>;
+
+  /**
+   * In case this Object schema is that of a reverse lookup column, store the info to link it back to the source column.
+   * Currently only used for Crossdoc
+   * @hidden
+   */
+  reverseLookupInfo?: {
+    sourceGridId: string;
+    sourceColumnId: string;
+    isStandard: boolean;
```

Comment:
It doesn't look like you used these in the `coda` PR? Having grid and column ids in the SDK isn't a great interface because those aren't concepts that exist in the SDK and it's hard to see how they could ever be used by a pack (as opposed to, say, slider mins and maxes which are useful to cross-doc as well as packs).

Can you just have `isReverseLookup?: boolean;` which seems like it would be sufficient for the `coda` PR?

Reply:
In order to simulate reverse lookups in a child doc we need the full set of information. The first PR is just disabling the feature while we build proper support.